locales/README.sv_SE.md

```diff
@@ -11,31 +11,31 @@
 
 ## Om
 
-Rocky Linux är ett Community Enterprise Operativ system designat att vara 100% bugg-för-bugg-kompatibelt med Enterprise Linux, nu när CentOS har ändrat inriktning.
+Rocky Linux är ett Community Enterprise operativsystem, designat att vara 100% bugg-för-bugg-kompatibelt med Enterprise Linux, nu när CentOS har ändrat inriktning.
 
 ## Vanliga frågor och svar
 > **F:** Vad menar ni med att, "CentOS har ändrat inriktning"
 
-CentOS-projektet tillkännagav nyligen ett skift i inriktning för CentOS. Medan CentOS tidigare fanns som ett nedströmsbygge av sitt uppströmsbygge (det som tar emot korrigeringar och uppdateringar efter att uppströmsleverantören gör det) kommer att flyttas till endast uppströmsbygge  (testade korrigeringar och uppdateringar innan de inkluderas i uppströmsbygget).
-Additionally, support for CentOS Linux 8 has been cut short, from May 31, 2029 to December 31, 2021.
+CentOS-projektet tillkännagav nyligen ett skift i inriktning för CentOS. Medan CentOS tidigare fanns som ett nedströmsbygge som tog emot korrigeringar och uppdateringar efter att uppströmsbygget implementerat dem, har CentOS-projektet skiftat till ett uppströmsbygge. Detta innebär att CentOS-projektet kommer att implementera korrigeringar och uppdateringar tidigare än sitt uppströmsbygge.  
```

Comment:
```suggestion
CentOS-projektet tillkännagav nyligen ett skifte i inriktning för CentOS. Medan CentOS tidigare fanns som ett nedströmsbygge som tog emot korrigeringar och uppdateringar efter att uppströmsbygget implementerat dem, har CentOS-projektet skiftat till ett uppströmsbygge. Detta innebär att CentOS-projektet kommer att implementera korrigeringar och uppdateringar tidigare än sitt uppströmsbygge.  
```

I know this is not a part of your PR but it might be worth changing 🙂